src/dotnet/APIView/APIViewWeb/DTOs/CommentUpdatesDto.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text.Json.Serialization;
+using APIViewWeb.LeanModels;
+
+namespace APIViewWeb.DTOs
+{
+    public enum CommentThreadUpdateAction
+    {
+        CommentCreated = 0,
+        CommentTextUpdate,
+        CommentResolved,
+        CommentUnResolved,
+        CommentUpVoteToggled,
+        CommentDeleted
+    }
+
+    public class CommentUpdatesDto
+    {
+        [JsonPropertyName("commentThreadUpdateAction")]
+        public CommentThreadUpdateAction CommentThreadUpdateAction { get; set; }
+        [JsonPropertyName("nodeId")]
+        public string NodeId { get; set; }
+        [JsonPropertyName("nodeIdHashed")]
+        public string NodeIdHashed { get; set; }
+        [JsonPropertyName("reviewId")]
+        public string ReviewId { get; set; }
+        [JsonPropertyName("revisionId")]
+        public string RevisionId { get; set; }
+        [JsonPropertyName("commentId")]
+        public string CommentId { get; set; }
+        [JsonPropertyName("elementId")]
+        public string ElementId { get; set; }
```

Comment:
Is this the line ID? If not what's the difference between node ID, comment ID and Element ID.

Reply:
`nodeId` and `elementId` is really the same thing. But this is really not a model just doing data transfer only some of the properties are used in different cases.